src/snakemake/settings/types.py

```diff
@@ -403,12 +403,11 @@ class OutputSettings(SettingsBase, OutputSettingsLoggerInterface):
     keep_logger: bool = False
     stdout: bool = False
     benchmark_extended: bool = False
-    log_errors_only: bool = (
-        False  # Only log errors. Sets log level to ERROR. Typically used for subprocces mode.
-    )
-    enable_file_logging: bool = (
-        True  # Create and write to logfile. Generally False for remote/subprocess executors.
+    log_level_override: Optional[int] = (
+        None  # Override log level (e.g., ERROR for subprocess)
     )
+    skip_plugin_handlers: bool = False  # Skip plugins/queue (remote mode)
+    enable_file_logging: bool = True
```

Comment:
If you would keep the old log_errors_only as well, then we can avoid a breaking change in the API here.

Reply:
Wrong, log_errors_only was only an intermediate new attribute in this PR. No breaking change here.